.luacheckrc

```diff
@@ -0,0 +1,36 @@
+stds.lpeg = require "lpeg"
+globals = { "_A", "_S", "action_queue_leave_bench", "class", "compare_tables",
```

Comment:
would be better to add this in the first commit, so new files are all added at the same time